yield_plugin/include/yield_plugin/yield_plugin_node.h

```diff
@@ -77,7 +77,7 @@ class YieldPluginNode
     
     worker.set_lanechange_status_publisher(lc_status_pub);
 
-    ros::ServiceServer trajectory_srv_ = nh.advertiseService("plugins/Yieldlugin/plan_trajectory",
+    ros::ServiceServer trajectory_srv_ = nh.advertiseService("plugins/YieldPlugin/plan_trajectory",
```

Comment:
Ah, good catch. This is the kind of thing that hopefully refactoring this boilerplate into a base class can solve for us, that these service creations/etc can be automatically defined for us.